src/components/EmptyStates/ActivityTabUpsell.js

```diff
@@ -20,14 +20,14 @@ const ActivityTabUpsell = () => (
         Get more with Find it Fix it capabilities
       </Title>
       <EmptyStateBody>
-        Upgrade to Red Hat Smart Management to remediate all your systems,
-        across regions and geographies directly from Red Hat Insights.
+        Upgrade to Satellite to remediate all your systems, across regions and
```

Comment:
Shouldn't Satellite be prepended with "Red Hat" as it was the case with "Red Hat Smart Management" previously?

Reply:
Good catch, that does sound right